check_tv.py

```
@@ -646,75 +738,80 @@ def edit_rtmp_key(driver, what):
-            logging.info("failed shutdown script")
-            os.system('TASKKILL /f /im ' + ffmpeg)
-            os.system('start relive_tv.py ' + text)
-            exit()
-
+      countshit = 0
+      MAX_RETRIES = 10  # example upper bound
+      while True:
+            try:
+                  print(url)
+                  streams = streamlink.streams(url)
+                  hls_stream = streams["best"]
```

Ruff (0.8.2)

Local variable `hls_stream` is assigned to but never used

Remove assignment to unused variable `hls_stream`

(F841)